spec/fixtures/actual/dir/legacy.ts

```diff
@@ -0,0 +1,7 @@
+import legacy from './legacy';
+import * as legacy2 from './legacy.js';
+import constFile from './const-file.ts';
```

Comment:
Why is it not `import { constFile } from "./const-file";` ?

Reply:
not sure it matters? just testing that the import for that line does not convert

Reply:
Yes, but TypeScript linter shows an error. 
Will the file `.\fixtures\expect\di \legacy.js` be used in NodeJs or in typescript files?

Reply:
probably, but the syntax isn't under test :)